README.md

```diff
@@ -5,54 +5,35 @@
 [![](https://img.shields.io/badge/freenode-%23ipfs-blue.svg?style=flat-square)](https://webchat.freenode.net/?channels=%23ipfs)
 [![](https://img.shields.io/badge/readme%20style-standard-brightgreen.svg?style=flat-square)](https://github.com/RichardLitt/standard-readme)
 
-> Compact self-describing codecs. Save space by using predefined multicodec tables.
+> Save space by using the predefined multicodec table
 
 ## Table of Contents
 
 - [Motivation](#motivation)
-- [How does it work? - Protocol Description](#how-does-it-work---protocol-description)
+- [Description](#description)
+- [Examples](#examples)
 - [Multicodec table](#multicodec-table)
+  - [Adding new multicodecs to the table](#adding-new-multicodecs-to-the-table)
 - [Implementations](#implementations)
 - [FAQ](#faq)
-- [Maintainers](#maintainers)
 - [Contribute](#contribute)
 - [License](#license)
 
 ## Motivation
 
-[Multistreams](https://github.com/multiformats/multistream) are self-describing protocol/encoding streams. Multicodec uses an agreed-upon "protocol table". It is designed for use in short strings, such as keys or identifiers (i.e [CID](https://github.com/ipld/cid)).
+Multicodec is an agreed-upon codec table. It is designed for use in binary representations, such as keys or identifiers (i.e [CID](https://github.com/ipld/cid)).
```

Comment:
What's the definition of a "codec"? We're using it in an unconventional way. Usually the codec is a device or software that performs encoding and decoding.

Reply:
In the case of IPLD that’s exactly what codecs do, they encode and decode blocks.

Reply:
@Stebalien Do I recall correctly that you had a good answer to the question, why it is "codec" and not "content type". If yes, would you mind adding that as a FAQ?

Reply:
content type has an existing meaning, and if you use that meaning then you could apply multiple content types to many codecs, and multiple codecs to a single content type, which means they cannot be synonymous.

Example: A block of `dag-json` can also be interpreted using `json` and the semantic meaning of the links would b stripped. The “content type” is still JSON. Similarly, you could interpret as `raw` and lose all semantic meaning. Content type is essentially what we’ve tried to describe as a “format” but unfortunately there’s a lot of older libraries and documentation that use the term format interchangeably with codec.

To add the confusion if we were to re-use content type, we already need to apply content types in UnixFS but we have to apply them to a “file” which is actually a dag connecting many blocks of raw binary data. Nowhere in that graph would you be able to use a content-type in place of the codec as a correct signal, it has to be added to the File metadata, or potentially wrapped in content-type metadata (the UnixFSv1 approach).

In short, a codec tells you how to decode block data into IPLD Data Model (JSON types + Links) and a content type tells an application how to interpret data (which may or may not involve parsing). There’s certainly some overlap, video file content types signal a literal codec for interpretation, but our interpretation is limited to how to interpret a single block into Data Model.